scammer-label-propagation/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Build stage: compile Python dependencies
+FROM python:3.9-alpine as builder
```

Comment:
IIRC, alphine builds python packages from source instead of using wheels, so it might take a long time to build docker images with dependencies like pandas. 

How's the build time so far?
It took forever for one of my ML models, so I used a different base image
https://github.com/forta-network/starter-kits/blob/main/malicious-smart-contract-ml-py/Dockerfile